manifests/input.yaml

```diff
@@ -2,778 +2,208 @@ format: yamanifest
 version: 1.0
 ---
 work/INPUT/rmp_jra55_cice_1st_conserve.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_20201102/common_025deg_jra55/rmp_jra55_cice_1st_conserve.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/remapping_weights/JRA55/global.025deg/2023.05.15/rmp_jra55_cice_1st_conserve.nc
```

Comment:
The hashes are changing for the ice remapping weights files. 

Is this expected?

Reply:
This looks correct:

`a5ab556badacc038c034ce088745a511  /g/data/ik11/inputs/access-om2/input_20230515_025deg_topog/common_025deg_jra55/rmp_jra55_cice_1st_conserve.nc`